scala/private/macros/setup_scala_toolchain.bzl

```diff
@@ -8,13 +8,16 @@ def setup_scala_toolchain(
         scala_macro_classpath,
         scala_xml_deps = None,
         parser_combinators_deps = None,
+        scala_semanticdb_dep = None,
```

Comment:
please name it in plural form without scala prefix - `semanticdb_deps`

Reply:
@liucijus, I had used the singular here because it needs one-and-only one jar. But I can change it to be plural if you still want. (I beleive I spit an error if there are more than one jar in the dep list). Just let me know.

Reply:
@crt-31 deps toolchains are meant for multiple deps indirection. It's uncommon for people to need to specify multiple deps, but I think there's no need to remove such possibility, or am I missing something? Also I would like to keep things consistently named.

Reply:
OK. 
Using singular in the code now.